code/modules/response_team/ert_outfits.dm

```diff
@@ -494,7 +494,7 @@
 	l_pocket = /obj/item/melee/classic_baton/telescopic
 	backpack_contents = list(
 		/obj/item/grenade/chem_grenade/antiweed = 2,
-		/obj/item/reagent_containers/spray/cleaner/advanced = 1,
```

Comment:
Why exchange the cleaner for a light replacer, instead of just adding the light replacer? If this is your intention, the removal needs to be listed in the changelog.